infra/core/host/appservice.bicep

```diff
@@ -66,6 +71,7 @@ resource appService 'Microsoft.Web/sites@2022-03-01' = {
       {
         SCM_DO_BUILD_DURING_DEPLOYMENT: string(scmDoBuildDuringDeployment)
         ENABLE_ORYX_BUILD: string(enableOryxBuild)
+        AZURE_TRANSLATE_API_KEY : (!(empty(congitiveServiceName))) ? cognitiveService.listKeys().key1 : ''
```

Comment:
congitiveServiceName typo,
but could you move this into main.bicep app settings instead? I try not to change the modules in core

Reply:
It won't work..
To be able to access the key I had to use resource 'Microsoft.CognitiveServices/accounts@2023-05-01' which won't work for the scope of the main bicep so, I made it optional inside the app core.

Reply:
Per my other comment, does this example help? https://github.com/jongio/chat-with-your-data-solution-accelerator/pull/2/files#diff-7dc4bd7866595a4b8b38b2fe17f1770bd7c2e969867143dae801a3f2e673846aR21

Reply:
Yes, It does. Will work on the KV.